features/support/object_model_pages/admin/edit_aptc.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+#admin/families tab/ edit aptc action
+class EditAptc
+
+  def self.csr_pct_as_integer
+    '[data-aptc-cucumber="csr_pct"]'
```

Comment:
let's use `data-cuke` or `data-cucumber` for consistency

Reply:
Alright, didn't realize we were going for a universal testing attribute but that makes sense.